### PR TITLE
Hardening/1368 cppcheck parse args

### PR DESCRIPTION
**Pull request by kzangeli**
Style guide checks incorporated in make targets `di` and `release`.
For now, only `logMsg` and `parseArgs` libraries are included in the style checks.

### CODE REVIEW

File: scripts/style_check_in_makefile.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    cat LINT_ERRORS
+    echo "================================================================="
+    cat LINT | grep -v "Done processing"
+    echo 
+    exit 1
+  fi
+}
+
+style_check src/lib/logMsg
+style_check src/lib/parseArgs
```
Comment (fgalan) [inline scripts/style_check_in_makefile.sh]: I'd suggest to add now all the list of libraries, but in commented way. That way, we can have always an idea of the pending work.

File: scripts/cpplint.py [hunk 2]
```diff
@@ -2685,7 +2685,7 @@ def CheckLanguage(filename, clean_lines, linenum, file_extension, include_state,
   if Search(r'\bsprintf\b', line):
     error(filename, linenum, 'runtime/printf', 5,
           'Never use sprintf.  Use snprintf instead.')
-  match = Search(r'\b(strcpy|strcat)\b', line)
+  match = Search(r'\b(strcat)\b', line)  # KZ removed strcpy from the 'verboten list'
```
Comment (fgalan) [inline scripts/cpplint.py]: There are some comments about this, that are going to be noted in issue #1368

File: scripts/style_check_in_makefile.sh (hunk 1 above)
Comment (kzangeli) [inline scripts/style_check_in_makefile.sh]: Sure

File: scripts/cpplint.py (hunk 2 above)
Comment (fgalan) [inline scripts/cpplint.py]: Added to issue. NTC.

Comment (kzangeli) [inline scripts/cpplint.py]: Fixed in fc167ef

File: scripts/style_check_in_makefile.sh (hunk 1 above)
Comment (kzangeli) [inline scripts/style_check_in_makefile.sh]: Fixed in fc167ef

**Comment by fgalan** [comment]
LGTM

**Pull request merged**